package.json

```diff
@@ -13,6 +13,10 @@
           "types": "./index.d.ts",
           "default": "./dist/index.esm.js"
         },
+        "require": {
+          "types": "./index.d.cts",
+          "default": "./dist/index.cjs"
+        },
```

Comment:
I think the types key is also needed for the "default" case below, also pointing to "./index.d.cts"?

Reply:
I don't think so. When I try to add that key, It (VSCode) did not suggest adding that key.

Reply:
For now, everything is working without that type. But if you think we can add that too.

Reply:
Let's leave the types key out of this PR, as it doesn't seem to be needed to fix your problems. If someone can find a repro that shows we need it then I'll accept a PR then.